testing/testcafe/helpers/screenshot-comparer.ts

```diff
@@ -5,10 +5,6 @@ import * as path from 'path';
 import * as fs from 'fs';
 import { PNG } from 'pngjs';
 
-const testRoot = './testing/testcafe/';
-const screenshotsPath = path.join(testRoot, '/screenshots');
-const artifactsPath = path.join(testRoot, '/artifacts/compared-screenshots');
-
 const screenshotComparerDefault = {
   highlightColor: { r: 0xff, g: 0, b: 0xff },
```

Comment:
we can move default path to the `screenshotComparerDefault` object
```suggestion
  path: './testing',
  highlightColor: { r: 0xff, g: 0, b: 0xff },
```